docs/src/main/asciidoc/rest-client-reactive.adoc

```diff
@@ -697,6 +701,40 @@ by specifying `quarkus.rest-client.multipart-post-encoder-mode` in your
 clients created with the `@RegisterRestClient` annotation.
 All the available modes are described in the link:https://netty.io/4.1/api/io/netty/handler/codec/http/multipart/HttpPostRequestEncoder.EncoderMode.html[Netty documentation]
 
+=== Receiving Multipart Messages
+REST Client Reactive also supports receiving multipart messages.
+As with sending, to parse a multipart response, you need to create a class that describes the response data, e.g.
+
+[source,java]
+----
+class FormDto {
+    @FormParam("file")
```

Comment:
Though this is valid, I would use the reactive annotation here: `@RestForm`. The benefit is that we don't need to write the name `file` again, it will use the field name if it's blank.

Reply:
I'll use one on one field and the other on the second one and add info that they can be used interchangeably